godel_scan_analysis/src/godel_scan_analysis_node.cpp

```diff
@@ -0,0 +1,11 @@
+#include <ros/ros.h>
+
+#include "godel_scan_analysis/keyence_scan_server.h"
+
+int main(int argc, char** argv)
+{
+  ros::init(argc, argv, "scan_server");
+  godel_scan_analysis::ScanServer server("world_frame", "keyence_sensor_optical_frame");
```

Comment:
While we can remap, it's probably better to just make these settable through a parameter interface.